tests/ui/author/blocks.stdout

```diff
@@ -1,13 +1,23 @@
 if_chain! {
     if let ExprKind::Block(ref block) = expr.kind;
-    if block.stmts.len() == 2;
+    if block.stmts.len() == 3;
     if let StmtKind::Local(ref local) = block.stmts[0].kind;
     if let Some(ref init) = local.init;
     if let ExprKind::Lit(ref lit) = init.kind;
-    if let LitKind::Int(42, _) = lit.node;
+    if let LitKind::Int(42, ref suffix) = lit.node;
+    if let LitIntType::Signed(ref int_ty) = suffix
+    if let IntTy::I32 = int_ty
```

Comment:
```suggestion
    if let LitKind::Int(42, LitIntType::Signed(IntTy::I32)) = lit.node;
```

Reply:
Should the same be done with float suffixes?

Reply:
Yes I think so